runtime/src/accounts_db.rs

```diff
@@ -2965,10 +2965,9 @@ pub mod tests {
     }
 
     #[test]
-    fn test_accounts_purge_chained_purge_after_snapshot_restore_complex() {
```

Comment:
I understood this mystical test finally...

Reply:
Moreover, https://github.com/solana-labs/solana/pull/8811/files#diff-2099c5256db4eb5975c8834af38f6456R1497 made this previous form of this test fail.